contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -927,10 +932,10 @@ contract NodeOperatorsRegistry is AragonApp, Versioned {
 
         Packed64x4.Packed memory signingKeysStats = _loadOperatorSigningKeysStats(_nodeOperatorId);
 
-        stakingLimit = signingKeysStats.get(TOTAL_VETTED_KEYS_COUNT_OFFSET);
-        stoppedValidators = signingKeysStats.get(TOTAL_EXITED_KEYS_COUNT_OFFSET);
-        totalSigningKeys = signingKeysStats.get(TOTAL_KEYS_COUNT_OFFSET);
-        usedSigningKeys = signingKeysStats.get(TOTAL_DEPOSITED_KEYS_COUNT_OFFSET);
+        stakingLimit = uint64(signingKeysStats.get(TOTAL_VETTED_KEYS_COUNT_OFFSET));
+        stoppedValidators = uint64(signingKeysStats.get(TOTAL_EXITED_KEYS_COUNT_OFFSET));
```

Comment:
Do we want to save these names for the future?
I know that this method is from the past, though, changing names won't break the ABIs

Reply:
Updated the naming for returning variables

Reply:
awesome, ty :+1: